cmd/geth/retesteth.go

```diff
@@ -900,7 +898,8 @@ func retesteth(ctx *cli.Context) error {
 		log.Info("HTTP endpoint closed", "url", httpEndpoint)
 	}()
 
-	abortChan := make(chan os.Signal)
+	// the channel used with signal.Notify should be buffered
+	abortChan := make(chan os.Signal, 5)
```

Comment:
This change could be non-trivial if blocking is the _expected_ result of a send or recv. Are you sure this is safe?

Reply:
I did this because of https://staticcheck.io/docs/checks#SA1017
which seems reasonable

wdyt?

Reply:
👍